src/DynamoCoreWpf/Views/GuidedTour/GuideBackground.xaml.cs

```diff
@@ -60,6 +51,18 @@ private void RaisePropertyChanged(string info)
             }
         }
 
+        public GuideBackground(Window mainWindow)
```

Comment:
Please also add comments since this is public, you can just put `constructor`. Once that is done, all LGTM